deps/Makefile

```diff
@@ -1957,7 +2061,7 @@ $(GIT_SOURCE): $(BUILDDIR)/git-$(GIT_VER)/config.status
 $(BUILDDIR)/git-$(GIT_VER)/checked: $(GIT_SOURCE)
 	echo 1 > $@
 $(GIT_TARGET): $(GIT_SOURCE) $(BUILDDIR)/git-$(GIT_VER)/checked
-	$(call make-install,git-$(GIT_VER),NO_INSTALL_HARDLINKS=1 bindir="$(build_libexecdir)")
+	make install -C $(BUILDDIR)/git-$(GIT_VER) $(MAKE_COMMON) NO_INSTALL_HARDLINKS=1 bindir="$(build_libexecdir)"
```

Comment:
git not worth staging since we'll be getting rid of it soon, or was their build system not cooperating?